browser/extensions/brave_extensions_browser_api_provider.cc

```diff
@@ -8,7 +8,7 @@
 #include "extensions/browser/extension_function_registry.h"
 
 // TOOD: I don't know why this isn't automatically linked in
-#include "gen/brave/common/extensions/api/generated_api_registration.cc"
+#include "../gen/brave/common/extensions/api/generated_api_registration.cc"
```

Comment:
There was a compiling error with `#include <version>` because of a file in the root called version by having the root out path in the gen config, so I had to remove it and tweak includes like this